common.gypi

```diff
@@ -13,6 +13,7 @@
     'enable_pgo_generate%': '0',
     'enable_pgo_use%': '0',
     'python%': 'python',
+    'enable_lto': 'true',
```

Comment:
I believe this bypasses / overrides the `--enable-lto` configure flag?

Reply:
hmm, yeah this should just flip the flag in configure.py, at least flip it to true on known safe platforms.

Reply:
Actually I think this gets overridden by the `config.gypi` generated by `./configure`, so it's not actually enabling LTO

Reply:
I tried enabling it with --enable-lto and it in fact took longer than the current build, but it didn't take forever to link mkdsnapshot. It took longer but it finished. Am I missing something?

Reply:
It may be that you Mac is beefier than mine/your lld is newer than mine/you just have more patience than mine ;)